bitnami/matomo/Chart.yaml

```diff
@@ -30,4 +30,4 @@ name: matomo
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/matomo
   - https://www.matomo.org/
-version: 0.2.0
+version: 0.2.0
```

Comment:
Hi @mdhont, you missed updating the chart's version. Per the [initial commit](https://github.com/bitnami/charts/pull/11746/commits/2e8338677e8fe3a7003b7e5974e0986936251b50), a patch bump will suffice.